src/fastcs/transports/rest/rest.py

```diff
@@ -90,11 +90,9 @@ def _get_response_body(attribute: AttrR[DType_T]):
     )
 
 
-def _wrap_attr_get(
-    attribute: AttrR[DType_T],
-) -> Callable[[], Coroutine[Any, Any, Any]]:
-    async def attr_get() -> Any:  # Must be any as response_model is set
-        value = attribute.get()  # type: ignore
+def _wrap_attr_get(attribute: AttrR[DType_T]) -> Callable[[], Coroutine[Any, Any, Any]]:
+    async def attr_get() -> dict[str, object]:  # Must be any as response_model is set
```

Comment:
Nit: deprecated comment?

```suggestion
    async def attr_get() -> dict[str, object]:
```

Reply:
I think this is still valid, although perhaps it should say `object`